kolibri/plugins/coach/assets/src/views/coach-exercise-render-page/interaction-item.vue

```diff
@@ -1,24 +1,26 @@
 <template>
 
   <div class="attempt-box" :class="[selected ? 'selected' : 'non-selected']">
+    <template v-if="isAnswer">
+      <mat-svg
+        v-if="interaction.correct"
+        class="svg-item svg-correct"
+        category="action"
+        name="check_circle"
+      />
+      <mat-svg
+        v-if="!interaction.correct"
+        class="svg-item svg-wrong"
+        category="navigation"
+        name="cancel"
+      />
+    </template>
     <mat-svg
-      v-if="interaction.hinted"
+      v-else-if="isHint"
```

Comment:
It might be good to show errors here, because our default behaviour is to give the user credit and move on if there is an error.